Lombiq.HelpfulExtensions/Lombiq.HelpfulExtensions.csproj

```diff
@@ -60,6 +60,7 @@
     <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.3.0" />
   </ItemGroup>
 
+
```

Comment:
Guess this was mistakenly added.

Reply:
Rider keeps automatically adding this line whenever it parses the project file (e.g. every time I add a new file to the project). I got tired of always reverting it.